ccpage102v2.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Use a dictionary to store information about a person you know.
+# Store their first name, last name, age, and the city in which they live.
+# Print each piece of information stored in your dictionary.
+friend = {
+    'first_name': 'belina',
+    'last_name': 'smith',
+    'city': 'boston',
+}
+
+print(friend['first_name'].capitalize())
+print(friend['last_name'].capitalize())
+print(friend['city'].capitalize())
+
+
+# Use a dictionary to store people’s favorite numbers.
+# Think of five names, and use them as keys in your dictionary.
+# Think of a favorite number for each person, and store each as a value in your dictionary.
+# Print each person’s name and their favorite number.
+names_and_fav_numbers = {
+    'theo': 10,
+    'phil': 9,
+    'pam': 8,
+    'melinda': 7,
+    'sarah': 6,
+}
+
+# Option 1: Printing each person's name and number
+for name, number in names_and_fav_numbers.items():
+    print(f"\nName: {name.capitalize()}")
```

Comment:
I'd move this `\n` to the end of the following line.

Reply:
@paulghaddad I tried this suggestion, and I find that the spacing is off and not as clear.